.azure-pipelines.yml

```diff
@@ -0,0 +1,206 @@
+trigger: none # Do not build branches on push.
+pr:
+- master
+
+schedules:
+- cron: "0 0 * * *"
+  displayName: Nightly build
+  branches:
+    include:
+    - master
+  always: true # Also build if there were no code changes, because many of our tests rely on external infrastructure.
+
+variables:
+  BOWER_VERSION: '1.8.8'
+  BUNDLER_VERSION: '2.1.4'
+  COMPOSER_VERSION: '5.1.0' # The version refers to the installer, not to Composer.
+  CONAN_VERSION: '1.18.0'
+  FLUTTER_VERSION: 'v1.12.13+hotfix.9-stable'
+  GO_DEP_VERSION: '0.5.4'
+  PYTHON_PIPENV_VERSION: '2018.11.26'
+  RUST_VERSION: '1.35.0'
+  STACK_VERSION: '2.1.3.20190715'
+  VIRTUALENV_VERSION: '20.0.14'
+
+stages:
+- stage: build_test
+  displayName: Build and Test
+  jobs:
+  # TODO: Consider running the Docker job only if the Dockerfile was changed.
+  - job: 'Docker'
+    pool:
+      vmImage: 'ubuntu-18.04'
+    steps:
+    - task: Docker@2
+      inputs:
+        command: 'build'
+        Dockerfile: 'Dockerfile'
+        tags: |
+          ort
+    # TODO: Add a step to run ORT in the Docker container, e.g. the requirements task, to verify that the Docker
```

Comment:
The `Dockerfile` itself meanwhile runs `ort requirements` in the end, so I guess we can omit this.

Reply:
Removed the comment.